### ISSUE TITLE
Protoparse cannot parse default float values with only '.' at the end

### ISSUE THREAD

**Issue by pedgeio** [jhump/protoreflect#257]
Another edge case:

```proto
// b.proto

syntax = "proto2";

message Foo {
  optional float one = 1 [default = 100.];
  optional double two = 2 [default = 200.];
}
```

- `protoc b.proto -o /dev/null` completes successfully
- `goprotoc b.proto -o /dev/null` returns `b.proto:6:40: syntax error: unexpected '.', expecting ',' or ']'` (if you comment out the field `one`, it will also return an error on line 7, but you won't get both errors until #225 is done)

I verified that `100` and `100.0` work.

**Comment by jhump** [comment]
Interesting. I did not realize this was considered valid in protoc. I don't think this one is hard either though.

**Comment by jhump** [comment]
Fixed in #262
